en/controllers/request-response.rst

```diff
@@ -746,9 +746,7 @@ ics generated on the fly from a string::
 Callbacks can also return the body as a string::
```

Comment:
This section should be removed or updated to show how to use callback using `withBody()`.